README.md

```diff
@@ -1,89 +1,94 @@
 ![Paprika logo](https://user-images.githubusercontent.com/10501940/52080175-07327400-254c-11e9-9748-7a00f93a13a8.png)
 
-# Paprika
+<h1 align="center">Paprika</h1>
+<p align="center">Paprika is a react component library of the <a href="https://design.wegalvanize.com">Starling Design System</a>.</p>
```

Comment:
```suggestion
<p align="center">Paprika is the React component library of the <a href="https://design.wegalvanize.com">Starling Design System</a>.</p>
```
not sure about the article (it's not some library, it's _the_ library? )

Reply:
What about just
`React component library of the Starling Design System`

Reply:
We say Paprika above anyways

Reply:
works for me 👍